glados/lib/src/anys.dart

```diff
@@ -155,10 +156,10 @@ class ShrinkableList<T> implements Shrinkable<core.List<T>> {
 
 extension SetAyns on Any {
   Generator<core.Set<T>> setWithLengthInRange<T>(
-      core.int min, core.int max, Generator<T> itemGenerator) {
-    assert(min == null || min >= 0);
+      core.int min, core.int? max, Generator<T> itemGenerator) {
+    assert(min >= 0);
     return (random, size) {
-      final length = random.nextIntInRange(min, max);
+      final length = random.nextIntInRange(min, math.max(max ?? size, min + 1));
```

Comment:
I think `min` should also be nullable. That would make `any.listWithLengthInRange` and `any.setWithLengthInRange` have the same signature.

Reply:
I've made `any.listWithLengthInRange` and `any.setWithLengthInRange` have similar signatures and bodies.